app/assets/stylesheets/darkswarm/branding.css.scss

```diff
@@ -38,3 +38,9 @@ $light-grey: #ddd;
 $light-grey-transparency: rgba(0, 0, 0, .1);
 $black: #000;
 $white: #fff;
+
+// Zeplin colours
```

Comment:
I dont think we should refer to the spec doc in the code.
we can call these:
- something related to the places where they are used
- "2020 redesign"
- nothing - remove the comment

it looks like we are in need of some standard to organise the colours.
there's a base/colors file.
I see some of the colors start with clr, others dont.

Reply:
I've removed the comment.